src/main/java/smartthings/cassandra/CassandraConnection.java

```diff
@@ -172,13 +180,13 @@ public void setupMigration() {
 	public boolean migrationsTableExists() {
 		logger.debug("Checking for migrations table.");
 		if (cassandraVersion.startsWith("3.")) {
-			ResultSet existingMigration = execute("SELECT table_name " +
+			ResultSet existingMigration = executeWithQuorum("SELECT table_name " +
 							"FROM system_schema.tables	" +
 							"WHERE keyspace_name=? and table_name = 'migrations';",
```

Comment:
Also, another question. This migration picks up the default consistency level from application's yaml file which by default is "Local_quorum", right?

Reply:
AFAICS, the consistency level is set [here](https://github.com/SmartThingsOSS/cassandra-migration/pull/15/files#diff-0d110b466b4d9b8c4263831c889cb7aaL66), which is set to `ALL` and can't be overriden.